### PR TITLE
Adds a gradient overlay on the faculty carousel

### PR DESCRIPTION
**Pull request by roberthouse54**
#### What's this PR do?

This PR adds a gradient overlay to the faculty carousel tiles.

### CODE REVIEW

File: cms/templates/cms/home_page.html [hunk 1]
```diff
@@ -54,7 +54,7 @@
       <div class="banner-wrapper">
         <div class="banner-wrapper-content">
           <h1>
-            MITx MicroMasters: Learn now, pay later (or never)
+            MITx MicroMasters: Tagline Goes Here
```
Comment (aliceriot) [inline cms/templates/cms/home_page.html]: lol

File: cms/templates/cms/home_page.html [hunk 2]
```diff
@@ -66,12 +66,9 @@
...
           <div class="sub-banner-section">
             <div class="sub-banner-text">
-            <h3>What is MicroMasters?</h3>
-              <p>MicroMasters is a new digital credential for online learners.
-                The MicroMasters credential will be granted to learners who complete
-                an integrated set of graduate-level courses on edX.org and capstone exams.
-                With the MicroMasters credentials, learners can apply for an accelerated master’s
-                degree program on campus, at MIT or other top universities.</p>
+            <h3>What is MicroMasters Credential?</h3>
+              <p>MicroMasters is a professional and academic credential for online learners from anywhere in the world. Learners who pass an integrated set of MIT graduate-level courses on edX.org, and one or more proctored exams, will earn a MicroMasters credential from MITx, and can then apply for an accelerated, on campus, master’s degree program at MIT or other top universities.</p>
```
Comment (aliceriot) [inline cms/templates/cms/home_page.html]: this line is real long, could you break it at 100 characters or so?

File: cms/templates/cms/home_page.html [hunk 3]
```diff
@@ -121,7 +118,7 @@ <h2 class="quote-text">"We will find people who never thought they would be able
         </div>
         <div class="row quote">
           <img class="person" src="{% static 'images/sanjay1.png' %}" alt="">
-          <h2 class="quote-text">"Inverted admission has the potential to disrupt traditional modes of access to higher education... We’re democratizing access to a master’s program for learners worldwide."</h2>
+          <h2 class="quote-text">"Inverted admissions has the potential to disrupt traditional modes of access to higher education... We’re democratizing access to a master’s program for learners worldwide."</h2>
```
Comment (aliceriot) [inline cms/templates/cms/home_page.html]: another long line here

File: static/scss/program-page.scss [hunk 4]
```diff
@@ -220,6 +226,19 @@
...
+    .faculty-copy {
+      width: 100%;
+      height: 160px;
+      position: absolute;
+      bottom: -1px;
+      padding: 25px 22px;
+      z-index: 2;
+      background: -moz-linear-gradient(top,  rgba(0,0,0,0) 0%, rgba(0,0,0,1) 100%); /* FF3.6-15 */
+      background: -webkit-linear-gradient(top,  rgba(0,0,0,0) 0%,rgba(0,0,0,1) 100%); /* Chrome10-25,Safari5.1-6 */
+      background: linear-gradient(to bottom,  rgba(0,0,0,0) 0%,rgba(0,0,0,1) 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
```
Comment (aliceriot) [inline static/scss/program-page.scss]: there's a couple things that can automatically add all of these vendor prefixes, would it be helpful for us to set those up?

**Comment by roberthouse54** [comment]
I reverted the copy changes so this PR is now only to add a gradient overlay to the faculty carousel.

File: static/scss/program-page.scss (hunk 4 above)
Comment (roberthouse54) [inline static/scss/program-page.scss]: Let's do that in a later issue.

**Comment by aliceriot** [comment]
gradient looks good!

**Comment by aliceriot** [comment]
Would you mind just break up those two long lines? after that I think this is ready to merge

**Comment by roberthouse54** [comment]
Should be all set now @aliceriot

**Pull request merged**